steps/k8s/k8s.go

```diff
@@ -25,7 +26,7 @@ var _ = gauge.Step("Create cron job with schedule <schedule>", func(schedule str
 })
 
 var _ = gauge.Step("Delete cron job", func() {
-	k8s.DeleteCronJob(store.Clients(), store.GetScenarioData("cronjob"), store.Namespace())
+	k8s.DeleteCronJob(store.Clients(), store.GetScenarioData("cronjob"), store.Namespace()) //nolint
```

Comment:
please handle the error also in this place

Reply:
Done